MAUI/Calendar/calendar-identifier.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Calendar Identifier in .NET MAUI Calendar Control | Syncfusion
+description: Learn here all about the Calendar identifier in the Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Calendar identifier in .NET MAUI Calendar (SfCalendar)
+The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier using the [Identifier] property in the SfCalendar.
+
+<table>
+<tr>
+<th>Supported Calendars</th>
+<th>UnSupported Calendars</th>
+</tr>
+<tr>
+
+<td>
+* GregorianCalendar<br>
+* HijriCalendar<br>
+* KoreanCalendar<br>
+* PersianCalendar<br>
+* TaiwanCalendar<br>
+* ThaiCalendar<br>
+* UmAlQuraCalendar<br>
+</td>
+
+<td>
+* Gezer Calendar<br>
+* Haida Calendar<br>
+* Igbo calendar<br>
+* Islamic calendar<br>
+* Javanese calendar<br>
+* Maramataka (Māori lunar calendar)<br>
+* Nepal Sambat<br>
+* HebrewCalendar<br>
+* JulianCalendar<br>
+* JapaneseCalendar<br>
+* Yoruba calendar<br>
+</td>
+</tr>
+</table>
+
+N>
+* The Flow Direction will be updated based on the calendar identifier. All the DateTime values, such as DisplayDate, MinimumDate, MaximumDate, SelectedDate, SelectedDates, SelectedDateRange, SelectableDayPredicate, and SpecialDayPredicate, can be given in two ways.
+* The DateTime instance without specifying calendar identifier. The calendar will handle the DateTime value for the specified calendar identifier.
```

Comment:
Don't use calendar type - use calendar identfier

Reply:
Changed the calendar type to calendar identifier.